manifests-k8s/0000_namespace_openshift-special-resource-operator.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    control-plane: controller-manager
+    openshift.io/cluster-monitoring: "true"
+  name: openshift-special-resource-operator
```

Comment:
Do you want to add manifests-k8s to the PR? If yes, please rerun `PLATFORM=k8s make manifests` as they are not fully updated.